### PR TITLE
ACS Scanner use test refs

### PR DESCRIPTION
**Pull request by RTann**

### CODE REVIEW

File: ci-operator/config/stackrox/scanner/stackrox-scanner-master.yaml [hunk 1]
```diff
@@ -1,13 +1,15 @@
 base_images:
-  stackrox-test:
-    name: stackrox-test
-    namespace: stackrox
-    tag: latest
+  ubi:
```
Comment (RTann) [inline ci-operator/config/stackrox/scanner/stackrox-scanner-master.yaml:2]: not gonna lie, I don't know what this is for

**Comment by gavin-stackrox** [comment]
If I approve this it'll auto-merge and CI will start to fail because: https://github.com/stackrox/scanner/pull/778 is not merged yet.

**Comment by RTann** [comment]
@gavin-stackrox should be ready now, thanks

**Pull request merged**